posts/cocurrency-parallelism-in-elixir.md

```diff
@@ -0,0 +1,173 @@
+Exercises on Exercism are small, synthetic, and often seemingly trivial. It’s easy to imagine that experienced practitioners would have nothing to learn from them. However, solving these synthetic problems can push you to learn and apply parts of your language that you may not have explored. This new learning can lead you to solve real world problems more efficiently or in a more expressive way.
+
+[Parallel Letter Frequency](https://exercism.io/tracks/elixir/exercises/parallel-letter-frequency) is a medium difficulty exercise on [Exercism's Elixir Track](https://exercism.io/tracks/elixir) that unpacks a surprising number of interesting lessons. To solve this problem successfully, your solution should execute in parallel in a number of worker processes. Achieving true parallelism in Elixir is surprisingly easy compared to other languages, but if you've never written concurrent code in Elixir, it may seem a little daunting. One of the things you'll discover in solving this exercise is how easy Elixir makes it to write code that can execute concurrently or in parallel. Applying these skills to your own code can have a big impact on the performance of your applications.
+
+This exercise requires you to implement a `Frequency.frequency/2` function that determines the letter frequency in a list of strings. The calculation should be carried out in a number of worker processes, set by the `workers` argument:
+
+```elixir
+iex> Frequency.frequency(["Freude", "schöner", "Götterfunken"], workers)
+%{
+  "c" => 1, 
+  "d" => 1, 
+  "e" => 5, 
+  ...
+  "ö" => 2
+}
+```
+
+In this post we'll explore concurrency in Elixir by making a working sequential solution to this exercise concurrent. But before we jump into the code, let's take a moment to examine what "concurrency" and "parallelism" actually mean, and how to achieve both in Elixir vs other languages.
+
+## Concurrency and parallelism
```

Comment:
The additional parallelism vs concurrency discussion is ✨! It really helps bring the concept home!